packages/sdk/src/oracle/types/ethers.ts

```diff
@@ -3,4 +3,8 @@ export type { Signer, BigNumber, BigNumberish, Contract } from "ethers";
 export type { Overrides } from "@ethersproject/contracts";
 export { Provider, JsonRpcSigner, JsonRpcProvider, Web3Provider, FallbackProvider } from "@ethersproject/providers";
 export { TransactionRequest, TransactionReceipt, TransactionResponse } from "@ethersproject/abstract-provider";
+export { LogDescription } from "@ethersproject/abi";
 export type { Event };
+
+// taken from ethers code https://github.com/ethers-io/ethers.js/blob/master/packages/abi/src.ts/interface.ts#L654
+export type Log = { topics: Array<string>; data: string };
```

Comment:
Is this not importable from ethers?

Reply:
If you go to that line, you see its defined in line as a function parameter, so i just assumed it wasnh exported

Reply:
not pushing for a change in this instance, but in general a thing I like to do to extract types from libraries is using `Parameters<typeof someFn>` or `ReturnType<typeof someFn>`

Reply:
i think i can try that

Reply:
yea that works